tests/core/dom/documentfragment.js

```diff
@@ -209,6 +209,40 @@
 			assert.isTrue( inner2.equals( childNodesList.getItem( 1 ) ) );
 		},
 
+		// (#2021)
+		test_find: function() {
+			var frag = new CKEDITOR.dom.documentFragment( CKEDITOR.document ),
+				b1 = new CKEDITOR.dom.element( 'b' ),
+				b2 = new CKEDITOR.dom.element( 'b' ),
+				em = new CKEDITOR.dom.element( 'em' ),
+				found;
+
+			frag.append( b1 );
+			frag.append( b2 );
+			frag.append( em );
+
+			found = frag.find( 'b' );
+			assert.areEqual( 2, found.count() );
+			assert.isTrue( b1.equals( found.getItem( 0 ) ) );
+			assert.isTrue( b2.equals( found.getItem( 1 ) ) );
+		},
+
+		// (#2021)
+		test_findOne: function() {
+			var frag = new CKEDITOR.dom.documentFragment( CKEDITOR.document ),
+				b1 = new CKEDITOR.dom.element( 'b' ),
+				b2 = new CKEDITOR.dom.element( 'b' ),
+				em = new CKEDITOR.dom.element( 'em' ),
+				found;
+
+			frag.append( b1 );
+			frag.append( b2 );
+			frag.append( em );
+
+			found = frag.findOne( 'b' );
+			assert.isTrue( b1.equals( found ) );
```

Comment:
While `found` var makes perfect sense for the previous case, it is not needed here.

It could be just simplified down to

```js
assert.areEqual( b1, frag.findOne( 'b' ) );
```

Just a single line instead of three.